app/components/Dashboard/SimpleDepositWithdraw.jsx

```diff
@@ -56,26 +55,20 @@ class DepositWithdrawContent extends DecimalChecker {
             fee_asset_id:
                 ChainStore.assets_by_symbol.get(props.fee_asset_symbol) ||
                 "1.3.0",
-            feeStatus: {},
+            feeAmount: new Asset({amount: 0}),
```

Comment:
Please use only feeAsset, which contains itself amount and asset_id

Reply:
done